### PR TITLE
Scripts

### PR DESCRIPTION
**Pull request by rpavlik**
Here's what I ended up with in the process of getting ready for CI. Some scripts to help prep a build environment, some scripts to let you build in a single step (though I don't know how to package/distribute), and associated, plus a few additional, readme tweaks.

Also, I noted that while you said you had to do the command line build to get a release build, mine said debug when I followed your steps, so I did the old `--help` and found there was a `--configuration` parameter as expected, which is why the build scripts vary slightly from your build instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,21 @@ OSVR-Config is a utility to configure the OSVR Server, and gives you access to a
    * `npm install -g gulp` - Used by the prepublish script for frontend build tasks.
    * `npm install -g tsd` - (Optional) You may use this to install new TypeScript typings or update existing ones (not often used).
    * `npm install -g rimraf` (Optional) I only recommend this on Windows, and only with great caution. rimraf will delete directories recursively. It's only useful on Windows because it bypasses all the long file path issues introduced by `npm`'s `node_modules` folder. Please be careful - it does not prompt you for confirmation.
+   * Windows users: There's a script in `devtools` that will handle updating `npm` and installing just the required global tools for you.
 
 ## IDE support
+If you'd like to helo with development, and not just build the project:
```
Comment (JeroMiya) [inline README.md]: typo: "helo" -> "help"

File: devtools/win-install-coreclr.cmd [hunk 2]
```diff
@@ -0,0 +1,9 @@
+rem from https://docs.asp.net/en/latest/getting-started/installing-on-windows.html#install-asp-net-5-from-the-command-line
+
+rem install .NET version manager (DNVM)
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"
+
+rem install .NET execution environment (DNX) for coreclr
+dnvm upgrade -r coreclr
+
+pause
```
Comment (JeroMiya) [inline devtools/win-install-coreclr.cmd]: Should this have a newline at the end of the file?

**Comment by JeroMiya** [comment]
This looks good. Couple of nitpicky things (see comments). Thanks!

File: devtools/win-install-coreclr.cmd (hunk 2 above)
Comment (rpavlik) [inline devtools/win-install-coreclr.cmd]: does it not? sure enough, doesn't seem like it - wonder why github doesn't show that online for new files, only for patches that remove the newline at eof. Apparently I only edited this file with notepad2-mod, not atom. That's an easy fix.

**Pull request merged**